src/components/_pages/custom-oid/list/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { TableDataRow, TableHeader } from 'components/CustomTable';
+import { useCallback, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { actions, selectors } from 'ducks/oids';
+import { EntityType } from 'ducks/filters';
+import { Container } from 'reactstrap';
+import PagedList from 'components/PagedList/PagedList';
+import { ApiClients } from 'src/api';
+import { Link } from 'react-router';
+import { LockWidgetNameEnum } from 'types/user-interface';
+import { SearchRequestModel } from 'types/certificate';
+
+export default function CustomOIDList() {
+    const dispatch = useDispatch();
+
+    const oids = useSelector(selectors.oids);
+
+    const isDeleting = useSelector(selectors.isDeleting);
+    const isUpdating = useSelector(selectors.isUpdating);
+
+    const isBusy = isDeleting || isUpdating;
+
+    const oidsRowHeaders: TableHeader[] = useMemo(
+        () => [
+            {
+                content: 'OID',
+                sortable: true,
+                sort: 'asc',
+                id: 'oid',
+                width: '15%',
+            },
+            {
+                content: 'Display Name',
+                align: 'center',
+                sortable: true,
+                id: 'oidName',
+                width: '30%',
+            },
+            {
+                content: 'Description',
+                align: 'center',
+                sortable: true,
+                id: 'oidDescription',
+                width: '40%',
+            },
+            {
+                content: 'Category',
+                align: 'center',
+                sortable: true,
+                id: 'category',
+                width: '15%',
+            },
+        ],
+        [],
+    );
+    const onListCallback = useCallback(
+        (filters: SearchRequestModel) => {
+            return dispatch(actions.listOIDs(filters));
+        },
+        [dispatch],
+    );
+
+    const oidsList: TableDataRow[] = useMemo(() => {
+        return oids.map((oid) => ({
+            id: oid.oid,
+            columns: [<Link to={`./detail/${oid.oid}`}>{oid.oid}</Link>, oid.displayName || '', oid.description || '', oid.category || ''],
```

Comment:
It should use label from OidCategory enum to display in place of enum code of category